libraries/AP_Notify/NavioLED_I2C.h

```diff
@@ -17,10 +17,14 @@
 #pragma once
 
 #include "NavioLED.h"
+#include <AP_HAL/I2CDevice.h>
 
 class NavioLED_I2C : public NavioLED
```

Comment:
Thinking again about this driver. I think we could just replace with our newer ones based on RCOutputRGBLed. They have the benefit that they integrate well with changes to the motors rather than interfering with them.